indra/newview/llinventorymodelbackgroundfetch.cpp

```diff
@@ -254,35 +254,35 @@ bool LLInventoryModelBackgroundFetch::folderFetchActive() const
     return mFolderFetchActive;
 }
 
-void LLInventoryModelBackgroundFetch::addRequestAtFront(const LLUUID & id, bool recursive, bool is_category)
+void LLInventoryModelBackgroundFetch::addRequestAtFront(const LLUUID& id, bool recursive, bool is_category)
 {
     EFetchType recursion_type = recursive ? FT_RECURSIVE : FT_DEFAULT;
     if (is_category)
     {
-        mFetchFolderQueue.push_front(FetchQueueInfo(id, recursion_type, is_category));
+        mFetchFolderQueue.emplace_front(FetchQueueInfo(id, recursion_type, is_category));
```

Comment:
shouldn't all these `emplace()` variants usages avoid calling the constructor and just directly take the constructor's args?

Reply:
That definitely makes sense, yes. I updated the PR accordingly.